app/components/primer/aria_live.ts

```diff
@@ -0,0 +1,41 @@
+const safeDocument = typeof document === 'undefined' ? undefined : document
+
+// Announce an element's text to the screen reader.
+export function announceFromElement(el: HTMLElement, options?: {assertive?: boolean; element?: HTMLElement}) {
```

Comment:
cc. @joshblack can we use your [live_region_element ](https://github.com/primer/live-region-element/blob/main/packages/live-region-element/src/global-announce.ts#L126-L145)work here?

Reply:
@kendallgassner definitely! `@primer/live-region-element` should be available now in dotcom (I think we're managing it via the primer workspace). If you have any feedback would love to hear it 🙏 Also feel free to copy any of the code too if that's what you were referring to 👀

Reply:
Excellent news! I spoke to the team during our sync meeting on Monday and we decided to make the change to live-region-element in a follow-up PR.